### PR TITLE
Cast number inputs to a Number so that operations evaluate correctly

### PR DESCRIPTION
**Pull request by shefalijoshi**
**Author Checklist**

- Changes address original issue? Yes
- Unit tests included and/or updated with changes? Yes
- Command line build passes? Yes
- Changes have been smoke-tested? Yes

### CODE REVIEW

File: src/plugins/condition/utils/operations.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ export const OPERATIONS = [
     {
         name: 'between',
         operation: function (input) {
-            return input[0] > parseInt(input[1]) && input[0] < parseInt(input[2]);
+            return Number(input[0]) > Number(input[1]) && Number(input[0]) < Number(input[2]);
```
Comment (nikhilmandlik) [inline src/plugins/condition/utils/operations.js]: wondering need to handle case where user enters large number first and small number as second value.

File: src/plugins/condition/utils/operations.js [hunk 2]
```diff
@@ -88,7 +88,7 @@ export const OPERATIONS = [
     {
         name: 'notBetween',
         operation: function (input) {
-            return input[0] < parseInt(input[1]) || input[0] > parseInt(input[2]);
+            return Number(input[0]) < Number(input[1]) || Number(input[0]) > Number(input[2]);
```
Comment (nikhilmandlik) [inline src/plugins/condition/utils/operations.js]: same here

File: src/plugins/condition/utils/operations.js (hunk 1 above)
Comment (shefalijoshi) [reply src/plugins/condition/utils/operations.js]: Sure I can do that.

Comment (shefalijoshi) [reply src/plugins/condition/utils/operations.js]: Done.

File: src/plugins/condition/utils/operations.js (hunk 2 above)
Comment (shefalijoshi) [reply src/plugins/condition/utils/operations.js]: Done.

File: src/plugins/condition/utils/operations.js [hunk 3]
```diff
@@ -76,7 +76,12 @@ export const OPERATIONS = [
     {
         name: 'between',
         operation: function (input) {
-            return input[0] > parseInt(input[1]) && input[0] < parseInt(input[2]);
+            let numberInputs = [];
+            input.forEach(inputValue => numberInputs.push(Number(inputValue)));
+            let larger = Math.max(...numberInputs.slice(1,3));
+            let smaller = Math.min(...numberInputs.slice(1,3));
+            console.log(larger, smaller);
```
Comment (akhenry) [inline src/plugins/condition/utils/operations.js]: Let's remove this debugging output.

File: src/plugins/condition/utils/operations.js [hunk 4]
```diff
@@ -4,7 +4,7 @@ export const OPERATIONS = [
     {
         name: 'equalTo',
         operation: function (input) {
-            return input[0] === parseInt(input[1]);
+            return Number(input[0]) === Number(input[1]);
```
Comment (akhenry) [inline src/plugins/condition/utils/operations.js]: It seems as though one of these casts is unnecessary if the operation only applies to numbers?

File: src/plugins/condition/utils/operations.js [hunk 5]
```diff
@@ -76,7 +76,12 @@ export const OPERATIONS = [
     {
         name: 'between',
         operation: function (input) {
-            return input[0] > parseInt(input[1]) && input[0] < parseInt(input[2]);
+            let numberInputs = [];
+            input.forEach(inputValue => numberInputs.push(Number(inputValue)));
+            let larger = Math.max(...numberInputs.slice(1,3));
```
Comment (akhenry) [inline src/plugins/condition/utils/operations.js]: Nice, I like this approach. 👍

File: src/plugins/condition/utils/operations.js (hunk 3 above)
Comment (shefalijoshi) [reply src/plugins/condition/utils/operations.js]: Done.

File: src/plugins/condition/utils/operations.js [hunk 6]
```diff
@@ -85,10 +90,14 @@ export const OPERATIONS = [
             return ' is between ' + values.join(', ') + ' and ' + values[1];
         }
     },
-    {
+    {//45 is not between 100 and 89
```
Comment (akhenry) [inline src/plugins/condition/utils/operations.js]: Let's remove this comment

**Comment by akhenry** [comment]
@shefali Looks like there was a comment left in that shouldn't have been. Otherwise looks good.

File: src/plugins/condition/utils/operations.js (hunk 6 above)
Comment (shefalijoshi) [reply src/plugins/condition/utils/operations.js]: Sorry about that. I'll be more diligent about removing these going forward.

**Comment by akhenry** [comment]
## Reviewer Checklist

* Changes appear to address issue? Y
* Appropriate unit tests included? Y
* Code style and in-line documentation are appropriate? Y
* Commit messages meet standards? Y

**Pull request merged**